portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,15 +63,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   // Retrieves comments from datastore and converts them into Comment objects.
-  private ArrayList<Comment> getComments() {
+  private ArrayList<Comment> getComments(int numRequests) {
     Query query = new Query("Comment");
     PreparedQuery results = datastore.prepare(query);
 
     ArrayList<Comment> comments = new ArrayList<Comment>();
     for (Entity entity: results.asIterable()) {
```

Comment:
A while loop or for loop over a slice of the iterable might be more appropriate here, in general breaks are considered less readable and maintainable because when you start reading at the top of the loop, you assume that it will iterate over every element. A while loop tells you right from the beginning that you'll terminate under a certain condition.

Reply:
Makes sense--I'll figure out a way to do this.